UI/web-src/ui/styles/fonts.css

```diff
@@ -312,4 +312,9 @@
   src: local('Noto Sans Bold'), local('NotoSans-Bold'),
        url('../../fonts/noto-sans-v9-cyrillic-ext_cyrillic_latin-ext_devanagari_latin_greek_vietnamese_greek-ext-700.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
        url('../../fonts/noto-sans-v9-cyrillic-ext_cyrillic_latin-ext_devanagari_latin_greek_vietnamese_greek-ext-700.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+}
+
+@font-face {
+    font-family: 'Avenir';
+    src: url('../../fonts/avenir.otf'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
```

Comment:
What font weight is this? What languages are supported by this font?

Reply:
Also where is this font from? What is its license?

Reply:
Not sure, what do you mean with license but it was not bough by us and it can be found here https://www.fonts.com/font/linotype/avenir/45-book

Reply:
Font weight is 400, I added that

Reply:
@montoresdennis Please ask @greychrist about the font licensing. I am not sure how suitable otf fonts are for web fonts either, generally they are woff/woff2 format. We need to include fonts for all the weights that we plan on regularly using in the UI otherwise the fonts can get scaled weirdly AFAIK.

Reply:
@jazdw @montoresdennis the fonts were part of the PiTuitive marketing package provided by Mike and his designer (I forget her name).  They're the same as we used in Pi.   I will check with him to see if we have woff versions available and double-check the licensing.   Good catch Jared thank you.

Reply:
The fonts were downloaded for free from fontsgeek. https://fontsgeek.com/avenir-lt-std-font